README.adoc

```diff
@@ -72,15 +72,14 @@ mvn install liberty:start-server
 ```
 
 You can access two microservices to test their availability. The
-`http://localhost:9080/inventory/systems/localhost` microservice retrieves the information for a specific host. The
-`http://localhost:9080/inventory/systems` microservice retrieves the information for a list of all previously
+`http://localhost:9080/inventory/systems/localhost` URL retrieves the information for a specific host. The
+`http://localhost:9080/inventory/systems` URL retrieves the information for a list of all previously
 registered hosts.
 
-Point your browser to the MicroProfile Health Check endpoint at `http://localhost:9080/health`. From
-the endpoint, you can see the health of the two services as well as the overall health outcome of the
-application. One check shows the state of the `system`
-service, and the other shows the state of the `inventory` service. As you might expect,
-both services are in the `UP` state, and the overall outcome of the application is `UP`.
+Point to `http://localhost:9080/health` URL. From here, you can see the health of the two services as well as the overall
```

Comment:
=> Next, go to `http://localhost:9080/health` URL to see the health of the two services as well as the..

Reply:
Fixed